src/main/java/com/puppycrawl/tools/checkstyle/api/AbstractFileSetCheck.java

```diff
@@ -37,7 +38,7 @@
     implements FileSetCheck {
 
     /** Collects the error messages. */
-    private final LocalizedMessages messageCollector = new LocalizedMessages();
+    private final SortedSet<LocalizedMessage> errorMessages = new TreeSet<>();
```

Comment:
So we are going to rename once here and then rename again in another issue?
Why don't we just keep the same name here then?